### DISCUSSION TITLE
Does a Multi languages system exist?

### DISCUSSION THREAD

**Discussion by ShibaInu0413** [bongo-devs/lavamusic#168] [Q&A] [answered]
Multilingual system if not present
whether to turn him back on

**Comment by LucasB25** [comment]
https://github.com/brblacky/lavamusic/compare/1.0.6...1.0.7

it existed

**Comment by ShibaInu0413** [comment]
But he doesn't seem to exist anymore?
I didn't see him in 1.1.2

**Comment by LucasB25** [comment, marked as answer]
there is no longer what I could see

**Reply by ShibaInu0413** [reply]
What's the meaning
I don't see the existence of a multilingual system in the main branch

**Reply by LucasB25** [reply]
it existed but it no longer exists

**Reply by ShibaInu0413** [reply]
ok, i get it
I'm thinking about restarting this project

**Reply by ShibaInu0413** [reply]
I want him to have different languages in each server

**Reply by ShibaInu0413** [reply]
I am learning how to use i18n

**Reply by LucasB25** [reply]
good luck

**Reply by ShibaInu0413** [reply]
thank you
